routes/error-tracker.js

```diff
@@ -49,6 +47,39 @@ function ignoreMessageOrException(message, exception) {
 }
 
 /**
+ * @param {string} exception
+ * @return {string} standardized exception
+ */
+function stackTraceConversion(exception) {
+  const location = '([^ \\n]+):(\\d+):(\\d+)';
+  const chromeStackTraceRegex = new RegExp(`^\\s*at (.+ )?(?:${location}|\\(${location}\\))$`, 'gm');
+  const mozillaSafariStackTraceRegex = /^([^@\n]*)@(.+):(\d+):(\d+)$/gm;
+  let validException = '';
+  let match;
+  let validExceptions = [];
+  if (chromeStackTraceRegex.test(exception)) {
+    while ((match = chromeStackTraceRegex.exec(exception))) {
```

Comment:
Use `match()` since you're using the global flag.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match

> If the regular expression does not include the g flag, str.match() will return the same result as RegExp.exec().